app/master.js

```diff
@@ -123,43 +124,87 @@ class Master {
 
   /**
    * Returns a packet of various data about cpu usage and worker processes
-   * @return {object} with master property that contains various machine data
+   * @return {Promise<object>} with master property that contains various machine data
+   * @example
+   * // example resolved object:
+   * {
+   *  master: {
+   *    title: 'opentmi',
+   *    requireRestart: false,
+   *    pid: 1234,
+   *    memoryUsage: {rss: 123, heapTotal: 123, heapUsed: 123},
+   *    uptime: 100000, // [seconds]
+   *    coresUsed: '1 of 4'
+   *  },
+   *  hostname: 'opentmi-host',
+   *  os: 'linux',
+   *  osStats: {
```

Comment:
I would like to separate these "dynamic" values to behind own API, but lets continue with this for now..